cpp/src/io/comp/nvcomp_adapter.cpp

```diff
@@ -638,6 +687,7 @@ std::optional<size_t> compress_max_allowed_chunk_size(compression_type compressi
       CUDF_FAIL("Compression error: " +
                 nvcomp::is_compression_disabled(nvcomp::compression_type::ZSTD).value());
 #endif
+    case compression_type::LZ4: return 1 << 24;
```

Comment:
Should we write this in a way that is better aligned with the DEFLATE value (easier to read as 16 MB?)?
```suggestion
    case compression_type::LZ4: return 16 * 1024 * 1024;
```